### ISSUE TITLE
Half of the public instances are either not working at all or malfunctioning at best

### ISSUE THREAD

**Issue by pitsi** [cloudrac3r/bibliogram#104]
I discovered bibliogram ~2 months ago and I started using it via privacy redirect (browser extension). It has now been 1+ month that the main instance does not work at all and, as I was digging through the other public instances, I discovered that most of them have issues, so here goes.

The following were tested on today's brave (1.20.x, based on chromium 88), with google's dns and with a public profile as a reference (giannis_an34)
- bibliogram.art: leads to 502 bad gateway since mid January 2021
- bibliogram.snopyta.org: works as of today
- bibliogram.pussthecat.org: leads to 502 bad gateway
- bibliogram.nixnet.services: works as of today, although really slow on search
- bg.endl.site: leads to 500 that's an error
- bibliogram.13ad.de: leads to 403 forbidden
- bibliogram.pixelfed.uno: works as of today
- bibliogram.ethibox.fr: the page opens, but search returns a 503 blocked by instagram error
- bibliogram.hamster.dance: works as of today

Can you please fix the main one and remove the faulty ones from the instances list?

**Comment by pitsi** [comment]
Can I assume that the issue report was closed with no reasoning because the project has moved to sourcehut?
I am not willing to make another account there just for a single report.

**Comment by Perflyst** [comment]
That's the thing: Submit a new issue by sending an email to ~cadence/bibliogram-issues@todo.sr.ht with the subject in the subject line and the issue text as the email body.

You don't need an account
